benchmarks/operator_benchmark/pt/configs.py

```diff
@@ -0,0 +1,73 @@
+import operator_benchmark as op_bench
+
+"""
+Configs shared by multiple benchmarks
+"""
+
+# Configs for conv-1d ops
+conv_1d_configs_short = op_bench.config_list(
+    attr_names=[
+        'IC', 'OC', 'kernel', 'stride', 'N', 'L'
```

Comment:
We should add groups and padding here as well.

Reply:
sure thing.  This PR is a no-op for floating point, so not adding any new shapes here.  We can add new shapes in a future PR.